Documentation/CodeSnippets/TypoScript/Page.rst.txt

```diff
@@ -0,0 +1,13 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/TypoScript/Setup/Page.typoscript
+   :linenos:
+
+   page = PAGE
+   page {
+     typeNum = 0
+     // 10 = TEXT
```

Comment:
Why commented?

Reply:
"If you remove the comments :typoscript:`//` before line 4 and 5 there would be
an output of "Hello World!". But of course we want to output our Fluid template."